OpenEXR/IlmImf/ImfDwaCompressor.cpp

```diff
@@ -2820,7 +2820,7 @@ DwaCompressor::uncompress
                     //
                     // sanity check for buffer data lying within range
                     //
-                    if (cd->planarUncBufferEnd + dstScanlineSize - _planarUncBuffer[UNKNOWN] > _planarUncBufferSize[UNKNOWN] )
+                    if (cd->planarUncBufferEnd + (size_t) dstScanlineSize  > _planarUncBuffer[UNKNOWN] + _planarUncBufferSize[UNKNOWN] )
```

Comment:
static_cast. Also I'd suggest grouping the expressions on either side of the greater than within paranethesis, because more recent versions of clang will complain and warn with "are you sure?" style warnings.

Reply:
I can switch to static_cast... there were a few places in the file that are "(size_t)"... so I will change those over too for consistent styling.... at least the static_cast ones.  It does leave the file feeling a little "mixed style" since there is alot of pointer casting done the old style. Not sure the issue of casting style is best tackled in a separate file all the files. Let me know.

Reply:
done

Reply:
BTW the other "(size_t)" places in the file where casting from pointers to do a mask check for alignment... so I had to use reinterpret_cast.